monero-outreach-docs/translations/es/storyboard/story58.md

```diff
@@ -0,0 +1,67 @@
+
+
+
+
+## Bienvenidos Todos los Mineros
+
+Como el equipo de minería ASIC ha dominado el algoritmo de prueba-de-trabajo de Bitcoin, muchos mineros en-casa han sido dejados de lado. Esto deja la minería abierta sólo a unos pocos con mucho dinero y recursos para beneficiarse de la minería de Bitcoin. 
+
+Mientras que la minería ASIC se ha convertido en algo común para Bitcoin, Monero actualiza frecuentemente su algoritmo de prueba-de-trabajo para desalentar al equipo ASIC. Con este acercamiento, integrados CPU y GPU pueden permanecer competitivos y rentables. Esta causa ha sido y 
+seguirá siendo la promesa de Monero a la comunidad.
+
+La última mejora de núcleo de Monero incluye un algoritmo de punta Bulletproofs. La actualización mejora dramáticamente la privacidad, velocidad y costos de transacción de Monero. Todos estos nuevos beneficios harán que Monero destaque
+como una criptomoneda ideal como la adopción continúe mejorando. Las características dirigidas hacia la facilidad de transacciones no deben ser subestimadas.
+
+
+## Monero Pronto se Levantará 
+
+Con toda evidencia de que Monero implementa tecnología de punta, junto a una fuerte comunidad, creemos que la moneda XMR pronto se levantará como un gigante. Sobresaldrá de otras monedas que no resuelven nuevos problemas 
+y dependen fuertemente en marketing para ganar atención. Actualmente estamos viendo la caída de proyectos pobres al ver muchos [centros cambiarios que los eliminan](https://support.binance.com/hc/en-us/articles/360017720731) durante el declive del mercado. Muchas monedas lanzadas en el 2017 están ahora [totalmente en bancarrota](http://fortune.com/2018/02/25/cryptocurrency-ico-collapse/).
+
+Para estar seguros, los principios que guían a Monero son los mismos que lo mantienen saludable ahora y bien en generaciones futuras. ¡XMR está destinado a ser usado! No deberíamos olvidar que el libro blanco original de Bitcoin se propuso resolver el problema
+de moneda con reserva fraccionaria armada por bancos centrales en contra de sus ciudadanos. No suponía ser una simple roca para mascotas para mantener a salvo por siempre. 
+
+Una tecnología usable es una que perdura. Hemos visto derrotas en cripto mercados antes, incluso más largas y profundas que la actual. Pero hay algo diferente esta vez. Algunos de nosotros estamos finalmente despertando al 
+pobremente manejado control monetario que empeora las vidas de ciudadanos alrededor de ciudades y del mundo. Por ejemplo, la inflación ha dañado personas que desean guardar su valor. Espantosamente, el dólar americano ha perdido [96% de su poder de compra](https://comparegoldandsilverprices.com/news/economics-101/dollar-devaluation-since-1913/) desde 1913. Es más probable que haya una mayor inflación a nivel mundial a medida que gobiernos y bancos centrales se hunden en deuda y el crédito fácil.

```

Comment:
It must be necessary in some cases just remember that we could use a different word of translation according to a more structured sentence when translating. I was keeping the translation the more literally possible since these are concrete documents.

Reply:
Yeah just remember that literal translations might fall into reading errors or misunderstandings when reading the text. If you prefer not to change the words, you could just reorganize it a little so the meaning will have a better structure.